types/reach__router/reach__router-tests.tsx

```diff
@@ -40,7 +41,7 @@ render(
             {context => (
                 <>
                     <div>hostname is {context.location.hostname}</div>
-                    <button onClick={() => context.navigate('/')}>
+                    <button onClick={(): Promise<void> => context.navigate('/')}>
```

Comment:
This return annotation shouldn't be necessary

Reply:
A must, as this is a test to check the return value.